Applications/_Plugins/Common/d3dx12/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# =======================================================================
+# See https://github.com/microsoft/DirectX-Graphics-Samples/tree/master/Libraries/D3DX12
+#
+
+add_library(Plugin_Common_D3DX12 INTERFACE)
```

Comment:
I don't think this is code originally written in Compressonator. I think it should be treated as external code